src/main/java/com/fa/training/dto/AdvertisingContractDto.java

```diff
@@ -76,4 +76,15 @@ public AdvertisingContract dtoAsPersistent() {
 		return a;
 	}
 
+	public AdvertisingContractDto transformPersistentToDto(final AdvertisingContract contract) {
```

Comment:
I think you should be use the constructor for converting from entity to dto! What do you think about that?